docs/development/testing_continuous_integration.rst

```diff
@@ -117,48 +116,48 @@ of a commit to apply to a branch in order disable the deletion for the Focal sta
 
 .. code:: Diff
 
-    diff --git a/.circleci/config.yml b/.circleci/config.yml
-    index 4a9b0bd4c..d9aea01b8 100644
-    --- a/.circleci/config.yml
-    +++ b/.circleci/config.yml
-    @@ -354,13 +354,6 @@ jobs:
-               BASE_OS=focal make ci-go
-             no_output_timeout: 35m
-
-    -      - run:
-    -          name: Ensure environment torn down
-    -          # Always report true, since env should will destroyed already
-    -          # if all tests passed.
-    -          command: make ci-teardown || true
-    -          when: always
-    -
-         - store_test_results:
-             path: ~/sd/junit
-
-    diff --git a/devops/gce-nested/ci-go.sh b/devops/gce-nested/ci-go.sh
-    index 850324ecc..776120df4 100755
-    --- a/devops/gce-nested/ci-go.sh
-    +++ b/devops/gce-nested/ci-go.sh
-    @@ -16,4 +16,3 @@ export BASE_OS="${BASE_OS:-xenial}"
+   diff --git a/.circleci/config.yml b/.circleci/config.yml
```

Comment:
Tested that diff applies cleanly on current `develop`. A bit iffy to maintain this way in docs though, maybe we can script that in future?